source/introduction.rst

```diff
@@ -0,0 +1,90 @@
+.. SPDX-FileCopyrightText: 2020 Hyperledger
+   SPDX-License-Identifier: CC-BY-4.0
+
+   perun-doc documentation master file, created by
+   sphinx-quickstart on Thu May 17 17:20:50 2018.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+
+Introduction
+=============
+
+Perun
+-----
+
+Perun is an off-chain framework which allows users to make transactions without interacting with the blockchain. The main objective of the project is to make the blockchains ready for mass adoption and alleviate current technical challenges such as high fees, latency and low transaction throughput. Perun protocol can be used on top of any blockchain system to accelerate decentralized applications and lower transaction fees.
```

Comment:
Mention that it is based on research and cite research papers?

Reply:
[Done](https://github.com/hyperledger-labs/perun-doc/pull/26/commits/1930b733d9013d4623987a6a1cbd26147691e39e)